### ISSUE TITLE
CTM Crash with Adorn/Indium

### ISSUE THREAD

**Issue by bushtail** [PepperCode1/ConnectedTexturesMod-Fabric#14]
Stacktrace can be found [here](https://pastebin.com/VSEwQs2u).

Steps to reproduce: Gain coffee table in inventory - CTD (or in this case, crash to menu)

**Comment by bushtail** [comment]
CLOSED: fixed in new adorn update, i'm big dum dum

**Comment by bushtail** [comment]
Yeah, still crashes. Indium v 1.0.0, Sodium v 0.1.1+unknown, Adorn v 1.14.0+1.16.5

**Comment by bushtail** [comment]
connected textures v 0.4.0

**Comment by PepperCode1** [comment]
Can you check if the crash still happens without Indium and Sodium? If it doesn't crash this is either an incompatibility with Indium (though, I'm not sure how this could happen) or an bug in Indium.

**Comment by bushtail** [comment]
Yes, it does - it seems to be an incompatibility with Indigo itself rather than with Indium.

**Comment by PepperCode1** [comment]
Thanks, I'll check it out soon. Hopefully the next update will have this fixed.

**Comment by PepperCode1** [comment]
Please check if this bug is still happening in the 0.5.0 update. Thank you.

**Comment by Alkaline-Dreams** [comment]
Still occurs for me on 0.5
